src/components/linkButton/linkButton.module.css

```diff
@@ -72,9 +72,13 @@
 
 .secondary {
   /* Todo: this color needs to be changes */
```

Comment:
```suggestion
```

Reply:
Still think the variables here shouldn't be used in this way 🤔 Just match naming in another way